static/js/actions/ui.js

```diff
@@ -22,9 +23,12 @@ export const setDialogVisibility = (bool: boolean): Action => (
 
 // work history actions
 export const SET_WORK_HISTORY_EDIT = 'SET_WORK_HISTORY_EDIT';
-export const setWorkHistoryEdit = (bool: boolean): Action => (
-  { type: SET_WORK_HISTORY_EDIT, payload: bool }
-);
+export const setWorkHistoryEdit: Function = (bool: boolean): (d: Dispatch) => Promise => {
```

Comment:
`bool` is the argument name we use a lot elsewhere for other action creators, so I'd either want to stick with it or change all of them, and overall I'd probably prefer to just stick with the convention we've been using.